src/communication/network_interface_cl.c

```diff
@@ -7223,6 +7222,10 @@ qmgr_execute_query (const XASL_ID * xasl_id, QUERY_ID * query_idp, int dbval_cnt
       if (IS_QUERY_EXECUTE_WITH_COMMIT (flag))
 	{
 	  ptr = or_unpack_int (ptr, &end_query_result);
+	  if (end_query_result == NO_ERROR)
```

Comment:
It would be better to add a comment to explain the logic.

Reply:
I added comment by your opinion, how about it ?

Reply:
Good.